op-challenger/game/fault/test/claim_builder.go

```diff
@@ -33,34 +33,38 @@ func (c *ClaimBuilder) CorrectTraceProvider() types.TraceProvider {
 
 // CorrectClaim returns the canonical claim at a specified trace index
 func (c *ClaimBuilder) CorrectClaim(idx uint64) common.Hash {
-	value, err := c.correct.Get(context.Background(), idx)
+	pos := types.NewPosition(c.maxDepth, int(idx))
+	value, err := c.correct.Get(context.Background(), pos)
```

Comment:
This is actually only used internally to `ClaimBuilder` and ultimately from places that already have a `Position` (except `CreateRootClaim` but it can just use `types.NewPositionFromGIndex(1)` as the root position). So I'd probably just delete it and use `CorrectClaimAtPosition` instead.